CMakeLists.txt

```diff
@@ -260,7 +261,28 @@ if (Glib_FOUND)
 	include_directories(${Glib_INCLUDE_DIRS})
 	list(APPEND NAVIT_LIBS ${Glib_LIBRARIES})
 else(Glib_FOUND)
-	set_with_reason(support/glib "Glib not found" TRUE ${INTL_LIBS})
+	set(EXTERNAL_GLIB_SORT_VERSION 2.58)
+	set(EXTERNAL_GLIB_TAG 2.58.0)
+	ExternalProject_Add(project_glib
+		URL http://download.gnome.org/sources/glib/${EXTERNAL_GLIB_SORT_VERSION}/glib-${EXTERNAL_GLIB_TAG}.tar.xz
+		URL_HASH SHA256=c0f4ce0730b4f95c47b711613b5406a887c2ee13ea6d25930d72a4fa7fdb77f6
+		# Pulling from the repo forces us to use autogen and autoreconf
```

Comment:
What would be the problem with this?

Reply:
The issue is t that lmza is too old on the image and is unable to extract the xz as it is in a too new format apparently. That's why I was working on the wince image refactoring (the image is still not working as I was explaining in https://github.com/navit-gps/Dockerfiles/pull/4, so as long as it's not fixed, I cannot move forward on this PR.

Reply:
but why we cant use the git repo and compile with autoreconf and autogen?

Reply:
Because I didn't want to re-add autoconf & co back in the build process. Seems tedious.